miso-web/src/main/webapp/scripts/form_container.js

```diff
@@ -70,6 +70,17 @@ FormTarget.container = (function ($) {
                 });
               },
             },
+            {
+              title: "Sequencing Parameters",
```

Comment:
This does not belong on the container form because it's not a property of the container itself.